src/main/java/com/tieumy/exercise/StringEx06.java

```diff
@@ -0,0 +1,19 @@
+package com.tieumy.exercise;
+
+public class StringEx06 {
+
+    public String findVerifyNumber(final String barcode) {
+        char[] enteredBarcode = barcode.toCharArray();
+        int sumOfOdd = 0;
+        int sumOfEven = 0;
+        for (int i = 0; i < enteredBarcode.length; i++) {
+            if (i % 2 == 0) sumOfEven += Character.getNumericValue(enteredBarcode[i]);
+            if (i % 2 != 0) sumOfOdd += Character.getNumericValue(enteredBarcode[i]);
+        }
+        StringBuilder numbers = new StringBuilder();
+        for (int i = 0; i <= 9; i++) {
```

Comment:
Tính toán là ra mà, việc gì phải dùng vòng for

Reply:
DONE